Index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Somarib Portfolio</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Roboto&family=Roboto:400;700&subset=latin&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="stylesheet.css">
+
+    
+  </head>
+  <body>
+    <div class="outer-maincont">
+        <header class="app-bar">
+            <nav class="lower-bar">
+                <ul><li><a href="#" class="logoo">Somarib Emmanuel</a></li>
+                    <li><a href="#" id="mbm-icon"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li>
+                </ul>
+            </nav>
+        </header>
+
+      <main class="main-cont">
+        
+        <article class="header-section" id="head-section">
+            <div class="header-inner-cont">
+                <section class="inner-section">
+                    <h1 class="myname">Somarib Emmanuel</h1>
+                    <h2 class="jobtitle">Software Developer</h2>
+                    <p class="abilities">
+                          I can help you build a product , 
+                          feature or website Look through some of my work and experience! 
+                          If you like what you see and have a project you need coded,  
+                          don’t hestiate to contact me.
+                    </p>
+
+                </section>
```

Comment:
- I'll suggest you use a `div` or `article` tag here instead of a `section` tag so as to improve the HTML structure of your page. Please fix this 🙏